app/models/nearby_facility.rb

```diff
@@ -7,13 +7,44 @@ class << self
     attr_writer :validate_on_load
 
     def query(params)
-      json_from_file = File.read(Rails.root.join('modules', 'va_facilities', 'nearby.json'))
-      mocked_response = JSON.parse(json_from_file)
-      return mocked_response if params[:street_address] && params[:city] && params[:state] && params[:zip]
+      if params[:street_address] && params[:city] && params[:state] && params[:zip]
+        isochrone_response = request_isochrone(params)
+        return get_facilities_in_isochrone(params, isochrone_response)
+      end
 
       NearbyFacility.none
     end
 
+    def request_isochrone(params)
+      params[:drive_time] = '30' unless params[:drive_time]
+      address = "#{params[:street_address]} #{params[:city]} #{params[:state]} #{params[:zip]}"
+      query = {
+        waypoint: address,
+        maxtime: params[:drive_time],
+        timeUnit: 'minute',
+        dateTime: '07:30:00',
+        optimize: 'timeWithTraffic',
+        key: Settings.bing.key
+      }
+      response = Faraday.get "#{Settings.bing.base_api_url}/Isochrones", query
+      response.body
+    end
+
+    def get_facilities_in_isochrone(params, isochrone_response)
```

Comment:
I wanted to cut down on the use of the word `isochrone` in favor of the more general `polygon`, but because the response from Bing is transformed and passed to the PostGIS function `ST_MakePolygon` I felt like I should stick to `isochrone`.
![image](https://user-images.githubusercontent.com/12172511/58826293-0565e380-8606-11e9-965c-b5f70655d6cd.png)